src/connection.cpp

```diff
@@ -749,7 +711,7 @@ namespace oxen::quic
 
         ngtcp2_settings_default(&settings);
 
-        settings.initial_ts = get_timestamp();
+        settings.initial_ts = get_timestamp().count();
```

Comment:
`get_time` returns a steady_clock::time_point, this returns the .time_since_epoch() of that time_point.  (The main difference is that it now returns typesafe nanoseconds instead of a bare integer).